src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -207,4 +208,28 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('date')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('format')->defaultNull()->end()
+                        ->scalarNode('interval_format')->defaultNull()->end()
```

Comment:
Imho using `null` here makes more sense since then the default format is not modified [when calling setDateFormat()](https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Extension/Core.php#L52-54) in the configurator.